example/cumulus-tf/main.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.14.1"
+      version = "~> 5.0"
```

Comment:
Is there a reason to constrain it to v5.x? It would prevent breakages if e.g. v6 requires changes but maybe that's ok and preferable to manual updates?

Reply:
Yeah, understood on the risks. Just wondering if that approach generally is right. We can discuss outside of this PR.